### PR TITLE
manally generate the changes that ews would have

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Issue link
https://issues.redhat.com/browse/MGDAPI-2555

# What

# Verification steps

### CODE REVIEW

File: manifests/integreatly-rhsso/15.0.0/rhsso-operator.15.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+    name: sso75-rhel8-operator-7.5-annotation
+  - image: registry.redhat.io/rh-sso-7-tech-preview/sso75-rhel8-operator:7.5
+    name: rhsso-operator
+  - image: registry.redhat.io/rh-sso-7-tech-preview/sso75-init-container-rhel8:7.5
+    name: keycloak_init_container
+  - image: registry.redhat.io/rh-sso-7-tech-preview/sso75-init-container-rhel8:7.5
+    name: rhsso_init_container
+  - image: registry.redhat.io/rhel8/postgresql-10:1
+    name: postgresql
+  version: 15.0.0
```
[the remark this answers is not in the document]
Comment (austincunningham) [reply manifests/integreatly-rhsso/15.0.0/rhsso-operator.15.0.0.clusterserviceversion.yaml:375]: agree 
```suggestion
  version: 15.0.0
  replaces: keycloak-operator.v14.0.0
```

[the remark this answers is not in the document]
Comment (austincunningham) [reply manifests/integreatly-rhsso/15.0.0/rhsso-operator.15.0.0.clusterserviceversion.yaml:375]: https://github.com/integr8ly/integreatly-operator/pull/2081

**Comment by laurafitzgerald** [comment]
@austincunningham just as an fyi it's expected that the manifest check will fail until the image_mirror_mapping file is gone. This usually gets removed by the ews when the product goes GA.

**Pull request merged**